content/zh/_index.md

```diff
@@ -43,8 +43,7 @@ cascade:
 
 {{% blocks/lead color="white" %}}
 
-OpenTelemetry 是各类 API、SDK 和工具形成的集合。可用于仪表化、生成、采集和导出遥
```

Comment:
"仪表化" is not appropriate in this context. I rarely hear people using this word to describe the modification of software to generate and report telemetry or to make it capable of analysis, such as profiling.

Both "插装" and "插桩" could be better choices. Since "插桩" is widely used in the testing context, typically referring to the addition of a stub or a small piece of code that replaces another component during testing (it is also related to the concept of "mocking"), I recommend using "插装" in this context.